continuous_integration/install_spm12.sh

```diff
@@ -1,29 +1,38 @@
 #! /bin/bash
+# Time-stamp: <2017-01-17 13:42:00 cp983411>
 set -e
 
 # set -x  # echo on
 
-SPM_ROOT_DIR=~/opt/spm12   #  Installation directory
-
-SPM_SRC=spm12_r6685.zip
+SPM_URL="http://www.fil.ion.ucl.ac.uk/spm/download/restricted/utopia/"
+SPM12_FILE="spm12_r????.zip"
 MCRINST=MCRInstaller.bin
 
+SPM_ROOT_DIR=~/opt/spm12   #  local installation directory
+
+
 mkdir -p $SPM_ROOT_DIR && cd $SPM_ROOT_DIR
 
 if [ ! -d spm12 ]; then
-    if [ ! -f ${SPM_SRC} ]; then 
-	wget http://www.fil.ion.ucl.ac.uk/spm/download/restricted/utopia/${SPM_SRC}
+    if [ ! -f "${SPM12_FILE}" ]; then
+	wget --recursive --level=1 --no-directories --accept "${SPM12_FILE}" "${SPM_URL}"
```

Comment:
This will download everything, including html files, etc. Maybe filter only .sh, .zip and other pertinent extensions ?